content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md

```diff
@@ -0,0 +1,31 @@
+In May of 2017 a meetup was organized in Krakow, Poland. We announced it on the [XSF blog](https://xmpp.org/2017/04/first-krakow-xmpp-meetup/), and [on meetup.com](htpps://www.meetup.com/XMPP-Jabber-Messaging-Krakow/events/239508291/) (also a few social networks).  The reponse from the XMPP/Jabber community was very positive, as we were fresh off the heels of FOSDEM and many members had just recently met at the XMPP Summit.
+
+## Philosophy & goals
+
+Firstly, it was a cooperation between Tigase and MongooseIM. Both organizations provide open source XMPP server software and bring professional expertise on the protocol.  However, we wanted it to be a humble and fast organisation, rather than a meticulously planned event. Calling to mind the addage "done is better than perfect".  The ultimate goal of the meetup was to meet colleagues of the XMPP and Jabber community, brainstorm, and overall improve communication between users and developers of the protocol.  An additional benefit from the meeting so soon after the Summit was to strengthen the trend and popularity of XMPP meetups in the European Union.
+
+## Speakers and presentations
+
+Two speakers from Erlang Solutions and one from Tigase were on hand to provide presentations.
+
+Andrzej Wojcik of Tigase gave a presentation on "Why use XMPP for IoT?". A factual comparison: efficient and straight to the point as why other options do not provide a proper solution.  The alternatives, such as HTTP, AMQP, MQTT all have their advantages, but none bring a global solution as exhaustive as XMPP.  A demonstration was shown of a proof-of-concept of IoT devices being controlled VIA XMPP commands.
+
+![Andrzej Wojcik of Tigase](./Andrzej_Wojcik_Tigase.jpg)
+
+Szymon Mentel from Erlang Solutions, exposed the "ICE, STUN, and TURN" open standards implemented in MongooseICE, an open source server written in Elixir. The demo showed a movie and a camera stream from a Raspberry Pi.  The stream was provided using Mangosta Android software, an open source XMPP client.
+
+![Szymon Mentel from Erlang Solutions](./Szymon_Mentel_MongooseIM.jpg)
+
+Piotr Nosek from Erlang Solutions, interacted with the audience around "Fantastic XMPP use-cases", showing the near-complete feature coverage of the open standards protocol and exploring what's missing or could be improved.  We hope that this discussion let to participant brainstorming and even more use cases for the protocol.
+
+![Piotr Nosek from Erlang Solutions](./Piotr_Nosek_MongooseIM.jpg)
+
+## Feedback and outcome
+
+Nice feedback from all attendees, all were happy to attend. There were some interesting question and answers following the presentation.  Followup discussions indicated that participants were engauged and were eager to contribute to the session.  It’s clear that these meetups can be far more engaging than a traditional chat session.  Being physically present seems to bring more wild and bright ideas to the front.
```

Comment:
> Nice feedback from all attendees, all were happy to attend.

This is also a fragment; "we received nice feedback from all attendees; they were all happy to haev attended" maybe? Maybe not.

> engauged

engaged